pandas/_libs/tslib.pyx

```diff
@@ -1569,7 +1569,10 @@ cpdef convert_str_to_tsobject(object ts, object tz, object unit,
                 ts = obj.value
                 if tz is not None:
                     # shift for _localize_tso
-                    ts = tz_convert_single(ts, tz, 'UTC')
+                    #ts = tz_convert_single(ts, tz, 'UTC')
+                    ts = tz_localize_to_utc(np.array([ts], dtype='i8'), tz,
```

Comment:
hmm there as to be an error in ``tz_convert_single``. though it might simply be best to just replace that function to a call exactly like this, IOW (and see if anything else breaks)

```
cdef tz_convert_single(....)
   return tz_localize_to_utc(.....)[0]
```